simulation/game_dir/game.py

```diff
@@ -13,7 +16,13 @@ def __init__(self):
         self.__renderer = Renderer(self.__world)
 
     def run(self):
-        self.renderer.run_simulation()
+
+        # for i in range(1000):
+        event = threading.Event()
+        threading.Thread(target=self.world.update, args=(event,)).start()
```

Comment:
reverse this please simulation should run in the main thread and renderer should run in the secondary thread.